tests/z3/z3_test.cpp

```diff
@@ -5,11 +5,34 @@
 #include "z3_sort.h"
 #include "z3++.h"
```

Comment:
I don't think you need to include this since this test is no longer experimenting with the Z3 API. I would suggest renaming this to z3_test_sort.cpp or something. Also you probably want to add some asserts in here, like for the comparison method.